supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/PipelinesParameterConversions.scala

```diff
@@ -20,7 +21,11 @@ import scala.language.implicitConversions
 trait PipelinesParameterConversions {
   implicit val fileInputToParameter = new ToParameter[PipelinesApiFileInput] {
     override def toActions(fileInput: PipelinesApiFileInput, mounts: List[Mount], projectId: String) = {
```

Comment:
ToL existing but same as above re: List vs NonEmptyList

Reply:
Probably can turn this one into a `NeL` yep, even though eventually it'll be converted back to a `List` to give to the PAPI request.